update/genesis/export.go

```diff
@@ -317,6 +350,45 @@ func (se *stateExport) exportTx(key string, tx data.TransactionHandler) error {
 	return nil
 }
 
+func (se *stateExport) exportNodesSetupJson(validators map[uint32][]*state.ValidatorInfo) error {
+	acceptedListsForExport := []core.PeerType{core.EligibleList, core.WaitingList, core.JailedList}
+	initialNodes := make([]*sharding.InitialNode, 0)
+
+	for _, validatorsInShard := range validators {
+		for _, validator := range validatorsInShard {
+			if shouldExportValidator(validator, acceptedListsForExport) {
+				initialNodes = append(initialNodes, &sharding.InitialNode{
+					PubKey:        se.validatorPubKeyConverter.Encode(validator.GetPublicKey()),
+					Address:       se.addressPubKeyConverter.Encode(validator.GetRewardAddress()),
+					InitialRating: validator.GetRating(),
```

Comment:
GetRating gets the rating at the beginning of the epoch, GetTempRating is the rating updated after every round. Do you know which one we actually need?

Reply:
I guess that the temp rating then, but I cannot tell for sure

Reply:
it's ok, because at the start of epoch block the ratings are equal.